### PR TITLE
My Portfolio - Desktop Version

### PR DESCRIPTION
**Pull request by aamirkhan2478**
# My Portfolio

> I am using media queries in this branch and created a responsive website. This is for the desktop version and also use this website in future for my own portfolio.

## Built With

- HTML
- CSS

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -168,16 +185,23 @@ <h1 class="card-heading">Professional Art Printing Data</h1>
       </div>
     </section>
     <section class="about-section">
```
Comment (leonardodiasb) [inline index.html:277]: - [ ] Your 'About me' section has a lot of blank space at the top and bottom making it a little bit different from the Figma template. Kindly try to remove that blank space. Your design is really good!
Your design:
image
Figma design:
image

File: index.html [hunk 2]
```diff
@@ -59,23 +73,26 @@ <h1 class="work-main-heading">My Recent Works</h1>
...
+            <div class="btn-left">
+              <button type="button" class="btn-primary">See Project</button>
+            </div>
           </div>
         </div>
-        <div class="card">
+        <div class="card card-1">
           <h1 class="card-heading">Professional Art Printing Data</h1>
           <p class="card-text">
             A daily selection of privately personalized reads; no accounts or
```
Comment (leonardodiasb) [inline index.html]: - [ ] In the Figma template, the hover behavior of the regular project card is to make the title, description, tags, and opacity disappear and make the `See Project` button appears at the bottom of the card(look at the image below). Kindly try to implement that functionality so the user can have a view of the image under the information before clicking on the see project button.

Figma design:
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @aamirkhan2478 , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your CSS styling is great and it's very similar to Figma's!!!! ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL] Kindly try to use bullet points to describe the implemented changes in your PR description to make it easier for the code reviewers to understand.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (aamirkhan2478) [reply index.html]: Hi sir, I think it is not possible without js and I am just using CSS and only CSS it is not possible how can I implement this functionality?

File: index.html [hunk 3]
```diff
@@ -59,21 +73,104 @@ <h1 class="work-main-heading">My Recent Works</h1>
...
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <ul class="languages-list">
+              <li>css</li>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <div class="btn-left">
```
Comment (victorokoroji) [inline index.html:90]: - [ ] This project card is not showing on your User interface. Kindly fix it
image

File: assets/css/style.css [hunk 4]
```diff
@@ -542,113 +585,1066 @@ button:disabled {
...
   }
+
+  .card-desktop {
+    display: flex;
+    flex-direction: column;
+    margin-top: 10px;
+    margin-bottom: 20px;
+    justify-content: flex-end;
+    align-items: center;
+  }
```
Comment (victorokoroji) [inline assets/css/style.css:747]: - [ ] As for the hover effect on the cards suggested by the previous reviewer, i think it is possible to add this effect with pure CSS. The steps i would follow to achieve this is as follows:

- Increase the opacity of the images of the cards (You can use a filter CSS attribute to add opacity)
- use `display: none` to hide the see project button

When a user hovers a card,

- Use `display: none to hide the description and tags
- Remove the opacity
- use `display: block` to show the See project button. 

This is my thought process so you can have an idea of how to go about it.

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @aamirkhan2478 , 

Great work on making the changes requested by a previous reviewer 👏 🔥 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



## Top highlight 🎯

- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (aamirkhan2478) [reply index.html:90]: Hi @vickymarz, I don't know why it is not showing on your User interface. On my laptop and I also check for the other devices it's working fine here.

File: assets/css/style.css (hunk 4 above)
Comment (aamirkhan2478) [reply assets/css/style.css:747]: Hi @vickymarz, Thank you so much you really helped me it's working now. Once again thank you much.

**Review by Kandy-Peter — APPROVED** [review]
Hi @aamirkhan2478,

### APPROVED 🟢 

Great implementation of all the requirements in this milestone. 👏
Your web app looks really fantastic. 🎉 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

##  Optional suggestion
 
# CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**